src/components/select/months.js

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import * as R from "ramda";
+
+import Select from "react-select";
+import { monthMap } from "../constants";
+
+const getValue = (month) =>
+  Number(R.head(R.values(R.pick([month], R.invertObj(monthMap)))));
```

Comment:
I know I know I know this is gross, please give me another way and I'll take it.

Reply:
As an alternative, we could calculate `options` using vanilla JS like this, I think?

```js
const options = Object.entries(monthMap).map(([value, label]) => ({
  value: Number(value),
  label,
}));
```

FWIW, I've soured on Ramda pretty heavily as I feel like as much as I agree with a lot of the theory behind it, whenever I see it used in practice I'm always like, "WTF is going on why does this make me feel like an idiot." 🤷🏻‍♂️2️⃣¢